server/src/main/java/io/crate/expression/symbol/ScopedSymbol.java

```diff
@@ -56,10 +56,23 @@ public final class ScopedSymbol implements Symbol {
     private final ColumnIdent column;
     private final DataType<?> dataType;
 
+    /**
+     * Used to disambiguate columns with the same name.
+     **/
+    private final int identity;
+
     public ScopedSymbol(RelationName relation, ColumnIdent column, DataType<?> dataType) {
         this.relation = relation;
         this.column = column;
         this.dataType = dataType;
+        this.identity = 0;
+    }
+
```

Comment:
How about adding a comment like you had before to explain a bit the usage of this constructor?

Reply:
Added some docs

Reply:
Thx!